packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -8,6 +8,10 @@
     flex-direction: column;
     position: fixed;
 
+    //The query checks whether the browser is IE or not.
+    @media screen and (max-width: 768px) and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
I think we should just go ahead and add a max-width, regardless of the browser. Based on the styles that are present, I'm not sure why Chrome/FF/Safari are rendering the way they are.

@neilpfeiffer 49% is probably pretty close, but do you have any recommendations for a max-width for this?

Reply:
On second thought, I'm not sure why this is a max-width media query, as we want this to take effect when the viewport is larger than 768px.

I think we could do something like:
```scss
@import '~terra-breakpoints/lib/media-queries';

.notification-dialog {
  // This will ensure that by default, the notification dialog sizes to the viewport (minus the insets).
  width: calc(100% - 2 * var(--terra-notification-dialog-small-horizontal-inset, 10px));

  // This will ensure that above the medium breakpoint, a max-width is enforced
  @include terra-mq-medium-up {
    max-width: 49%; // or whatever Neil comes up with
  }
}
```

Reply:
Investigating to see what would be an appropriate width to set.

Reply:
@pranav300 Lets set max width to be a fixed `800px` to match one of our [disclosure widths](https://github.com/cerner/terra-framework/blob/1e9427e3312236e517ae8691fd64a6d9d109a0a4/packages/terra-disclosure-manager/src/DisclosureManager.jsx#L18) 

Which should be an appropriate max-width for the primary type of content included in the notification-dialog. If we find that is too small after hearing feedback, we can increase it to `960px`.